Gemfile

```diff
@@ -46,7 +46,9 @@ end
 group :test do
   # Adds support for Capybara system testing and selenium driver
   gem 'capybara', '>= 2.15'
+  gem 'rspec-rails', '~> 5.0.0'
```

Comment:
Just a tip:

If you plan to use Rspec you can create your application like this: `rails new my_app -T` so you don't have the standard minitest folder